src/BinSkim.Driver/DriverUtilities.cs

```diff
@@ -45,5 +49,35 @@ private static ConventionBuilder GetConventions<T>()
 
             return conventions;
         }
+
+        public static IList<string> GetInaccessibleSymbolPaths(string symbolPathString)
+        {
+            IList<string> inaccessibleList = new List<string>();
+            if (string.IsNullOrWhiteSpace(symbolPathString)) { return inaccessibleList; }
+
+            string[] symbolServers = Array.FindAll(symbolPathString.Split('*'), path =>
+            path.StartsWith("http://", StringComparison.OrdinalIgnoreCase) ||
+            path.StartsWith("https://", StringComparison.OrdinalIgnoreCase));
+
+            foreach (string symbolServer in symbolServers)
+            {
+                try
+                {
+                    var httpClient = new HttpClientWrapper();
+                    HttpResponseMessage httpResponseMessage = httpClient.GetAsync(symbolServer).GetAwaiter().GetResult();
```

Comment:
We are doing to an HTTP GET here: what is the authentication model? Binskim mostly runs in the context of Guardian---will that have sufficient privileges? Or will the majority of these checks result in an inaccessible error?